snafu/upgrade_openshift_wrapper/trigger_upgrade.py

```diff
@@ -161,7 +161,7 @@ def _run_upgrade(self):
         total_time = end_time - start_time
         logger.info("Total upgrade time: %s" % total_time)
 
-        return init_version, new_version, platform, start_time, end_time, total_time
+        return init_version, desired_version, new_version, platform, start_time, end_time, total_time
```

Comment:
This return statement was updated, but the return statement on line 119 was not. Can we update that one as well? Looks like it would need to be:

```python
return init_version, desired_version, desired_version, platform, the_time, the_time, (the_time - the_time)
```

Reply:
yes my bad, that was missed, corrected it now. Thanks